### DISCUSSION TITLE
macOS: Make menu bar icon optional and white

### DISCUSSION THREAD

**Discussion by Ahrdie** [rustdesk/rustdesk#7653] [Q&A] [answered]
It is common for applications to make the showing of a menubar icon optional, and it would be great, if rustdesk would make this configurable as well.
Furthermore it would fit in much better if it followed the praxis of having a white icon.
<img width="442" alt="menubar with multiple white icons and rustdesks black one" src="https://github.com/rustdesk/rustdesk/assets/3695677/83e77cf3-32db-41dc-992b-a5c08dc117b1">

**Comment by rustdesk** [comment, marked as answer]
Fixed in nightly build
